src/test/java/com/aws/greengrass/clientdevices/auth/certificate/BackgroundCertificateRefreshTest.java

```diff
@@ -82,13 +81,12 @@ public class BackgroundCertificateRefreshTest {
     @Mock
     private GreengrassV2DataClient client;
     @Mock
```

Comment:
The tests below were changed to reflect the functionality we actually wanted. Trigger the refresh when there is network connectivity after at least 24 hours